packages/styles/scss/components/leadspace/_leadspace.scss

```diff
@@ -0,0 +1,69 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../globals/imports';
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/grid/grid';
+
+@mixin leadspace {
+  .#{$prefix}--leadspace {
+    position: relative;
+    height: carbon--rem(560px);
+
+    &__image {
+      object-fit: cover;
+      width: 100%;
+      height: 100%;
+    }
+
+    &__content-container {
+      padding: carbon--mini-units(4) 0 carbon--mini-units(4)
+        carbon--mini-units(4);
+      position: absolute;
+      top: 0;
+      height: 100%;
+      width: 100%;
+    }
+
+    h1 {
+      @include carbon--type-style(display-01);
+      @include carbon--make-col(7, 16);
+    }
+
+    p {
+      @include carbon--type-style(expressive-heading-03);
```

Comment:
I think the expressive type you're using in this file is supposed to be fluid. I think in order to make that happen you need to pass in the `fluid` parameter. I think it's `false` by default.

```scss
@include carbon--type-style(expressive-heading-03, true);
```

I think this would also apply for [#L32](https://github.com/carbon-design-system/ibm-dotcom-library/pull/78/files#diff-581c7770913f29a42c27bb5d30476ce9R32) as well.